drheader/core.py

```diff
@@ -314,16 +314,19 @@ def __validate_rules(self, config, header, directive=None):
 
         if config['Required'] is True or (config['Required'] == 'Optional' and header in self.headers):
             if config['Enforce']:
-                self.__validate_rule_and_value(config['Value'], header, directive)
+                if "Severity" in config:
+                    self.__validate_rule_and_value(config['Value'], header, directive, config['Severity'])
+                else:
+                    self.__validate_rule_and_value(config['Value'], header, directive, config)
```

Comment:
You've specified a default value of `False` for `Severity` on line 121, but you're passing in the config here when the severity isn't provided, so the default value isn't being used. The report will end up showing something along the lines of:
```
{
    "rule": "Content-Security-Policy",
    "severity": {
        "Required": true,
        "Enforce": false,
        ...
    }
}
```
```suggestion
                    self.__validate_rule_and_value(config['Value'], header, directive)
```